### ISSUE TITLE
[4.0.0] Strings in CSS variables gets unquoted

### ISSUE THREAD

**Issue by ngyikp** [jakubpawlowicz/clean-css#867]
```
Ngs-Applejack:test ngyikp$ npm install clean-css clean-css-cli
test@1.0.0 /Users/ngyikp/Desktop/test
├── clean-css@4.0.0 
└─┬ clean-css-cli@4.0.0-rc0 
  └── clean-css@4.0.0-pre  (git://github.com/jakubpawlowicz/clean-css.git#fdf123b73405f330ee46b8fdc9dc240353a35e20)

Ngs-Applejack:test ngyikp$ ./node_modules/.bin/cleancss -o test.min.css test.css 
```

BTW, `clean-css-cli` should bump the dependency version and not rely on a commit hash...?

Input:
```css
#a:after {
	--test: "XYZ";
	content: var(--test);
}
```

Output:
```css
#a:after{--test:XYZ;content:var(--test)}
```

Running with `-O1 'removeQuotes:off'` is a workaround.

**Comment by jakubpawlowicz** [comment]
Thanks for bug report 👍 

CLI is not on 4.0 yet. Stay tuned.

**Comment by jakubpawlowicz** [comment]
CLI 4.0 is out too.

I'll deal with the bug tomorrow.

**Comment by jakubpawlowicz** [comment]
Variables unquoting fixed in 4.0.1.
